### PR TITLE
feat: react native in app components

### PR DESCRIPTION
**Pull request by matgar26**

### CODE REVIEW

**Comment by connorlindsey** [comment]
We use a tool to keep dependencies in sync between packages. You can run `yarn manypkg fix` to fix any discrepancies and that'll fix CI.

File: packages/react-native/src/theme/theme.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    knock0: 11,
+    knock1: 12,
+    knock2: 14,
+    knock3: 16,
+    knock4: 18,
+    knock5: 20,
+    knock6: 24,
+    knock7: 30,
+    knock8: 36,
+    knock9: 48,
```
Comment (cjbell) [inline packages/react-native/src/theme/theme.ts:27]: Nit: can we remove the `knock` prefix here please. This is already under our specific package so is superfluous

File: packages/react-native/src/modules/feed/components/NotificationFeedComponents/NotificationFeedCell/NotificationFeedCell.tsx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+                ]}
+              />
+              {resolvedStyle.showAvatarView && actor && isUser(actor) && (
+                <View>
+                  <AvatarView name={actor.name} src={actor.avatar} />
+                </View>
+              )}
+            </View>
+            <View style={layoutStyles.contentContainer}>
+              {item.blocks.map((block, index) => {
```
Comment (cjbell) [inline packages/react-native/src/modules/feed/components/NotificationFeedComponents/NotificationFeedCell/NotificationFeedCell.tsx:171]: I think one thing we're missing here right now is support for the `action_url` block, which is a text block that contains the action that should be invoked on tap of the notification cell.

**Review by cjbell — COMMENTED** [review]
this looks awesome matt! tiny comments

File: packages/client/src/clients/feed/feed.ts [hunk 3]
```diff
@@ -410,6 +410,53 @@ class Feed {
     return result;
   }
 
+  async markAllReadAsArchived() {
+    // Note: there is the potential for a race condition here because the bulk
+    // update is an async method, so if a new message comes in during this window before
+    // the update has been processed we'll effectively reset the `unseen_count` to be what it was.
+    const { items, ...state } = this.store.getState();
+    console.log("[Feed] Items", items);
```
Comment (cjbell) [inline packages/client/src/clients/feed/feed.ts:418]: 👀 can you remove all of these console.logs please

**Pull request merged**